### DISCUSSION TITLE
Is it possible to for the Autocomplete component to show images alongside the text entries

### DISCUSSION THREAD

**Discussion by itslinotlie** [skeletonlabs/skeleton#2745] [Q&A] [answered]
I looked at the available props for the `Autocomplete` component, and I think the closest thing would be the `regionItem` but I'm not sure if that's enough? Would be more than happy to be proved wrong though.

**Comment by endigo9740** [comment, marked as answer]
Hey @itslinotlie not currently in Skeleton v2, however the goal is to revisit and rethink this component in Skeleton v3 - to make it much more flexible and customizable. We've been actively encouraging folks to leave feature requests per each v3 milestone issue. There's already a related request in the Community Request section here, but you're welcome to leave a comment echoing support for this as well:

- https://github.com/skeletonlabs/skeleton/issues/2366

Also just FYI, this may not end up taking the form of a dedicating component. We recently announced our upcoming [Floating UI port for Svelte](https://github.com/skeletonlabs/floating-ui-svelte/discussions/50). As part of this we will have access to a dedicated Typeahead feature:

- React Docs: https://floating-ui.com/docs/useTypeahead
- Demo: https://codesandbox.io/p/sandbox/fragrant-water-bsuirj?file=%2Fsrc%2FApp.tsx

It's a different way to solve for this type of interface, but may end up being more appropriate and flexible.

But either way, between v3 or Floating UI, we will have a proper solution for this going forward. Hope that helps!
